docker-compose.yml

```diff
@@ -3,16 +3,24 @@ version: "3.7"
 services:
   app:
     build: ./
+    environment:
+      PORT: 3001
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: root
+      POSTGRES_HOST: postgres
+      POSTGRES_PORT: 5432
+      POSTGRES_DB: db_b_e_course
     ports: 
       - "3001:3001"
 
   postgres:
     image: postgres:latest
     environment:
-      POSTGRES_USER: "postgres"
-      POSTGRES_PASSWORD: "root"
-      POSTGRES_DB: "db_b_e_course"
-
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: root
+      POSTGRES_HOST: localhost
+      POSTGRES_PORT: 5432
+      POSTGRES_DB: db_b_e_course
```

Comment:
Why you did't use values from .env file, like `src/db/dbAuth.ts` ?

Reply:
@barantarasdev as we are developing course project, I decided to leave env variables in docker to make it easier to test project by our mentor with no need of additional files, thats it :)